### DISCUSSION TITLE
[Question] longhorn how to change xfs to ext4

### DISCUSSION THREAD

**Discussion by jiayu997** [longhorn/longhorn#8970] [General]
## Describe the bug
longhorn version: 1.5.3
in our someone project, longhorn fstype is xfs，but always occur xfs error, like xfs(sda): metadata I/O error in "xlog_iodone" .... , our test environment and production environment ext4 fstype has nerver been situation like this，we suspect that xfs filesystem is unstable cause this problem. can we adjust the existing pv file format from xfs to ext4

**Comment by derekbit** [comment]
Not sure if the xfs issue is related to https://github.com/longhorn/longhorn/issues/8236. You can try to upgrade the system to 1.5.5. Do you know how to reproduce the issue?

It is unable to convert a filesystem to another filesystem. You can
- copy the data out
- create a longhorn volume with ext4 filesystem
- copy the data into the volume

**Reply by jiahaoc1993** [reply]
When does the "trim" process occur? In my understanding, if we don't manually trim the Longhorn volume, issues related to #8236 should not occur.
